scripts/svgDtsTemplate.js

```diff
@@ -0,0 +1,16 @@
+function svgDtsTemplate({ types: t, template }, _, { componentName }) {
+  const typeScriptTpl = template.smart({ plugins: ['typescript'] })
+  const {
+    typeAnnotation,
+  } = typeScriptTpl.ast`type Foo = React.FunctionComponent<React.SVGProps<SVGSVGElement>>`
```

Comment:
Just doing this so i can easily grab the AST node for the type annotation and add it to the component identifier